ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c

```diff
@@ -2104,6 +2135,10 @@ bb_decode (void **pdata, const GstTensorsConfig * config,
     goto error_unmap;
   }
 
+  if (bdata->do_log != 0) {
```

Comment:
how about ```if (bdata->do_log)``` or ```if (bdata->do_log == 1)``` ?

Reply:
@songgot ```do_log``` is not a boolean type variable, meaning only 0 can represent FALSE.

Reply:
Okay, thank you.